docs/extend/modular-resources/create/_index.md

```diff
@@ -517,4 +517,6 @@ Your executable will be run by `viam-server` as root, so dependencies need to be
 
 ## Next steps
 
-Follow [these configuration instructions](/extend/modular-resources/configure/) to add your custom resource to your robot.
+Once you have created your custom resource, follow [these configuration instructions](/extend/modular-resources/configure/) to add the custom resource to your robot.
+
```

Comment:
Flipped! Also added top of page as well, good callout.

Happy to use new examples if provided them. There are two good spots for advanced examples I'd be happy to slot in.

`create` is engineering nomenclature, we cannot fix that clobbering of concept all the way downstream here in the docs. If we gain some internal eng comfort with a better internal eng left nav layout: I worry that we lose more external audience members who would be confused that `create` -- long the page for this task -- is now `program` which already is [its own concept.](https://docs.viam.com/program/).

Strong suggest that product and eng reconsider the naming of `viam module generate-metadata-for-existing-module` to something other than `viam module create`. I've used "generate metadata" language throughout to try to workaround. It will be misunderstood by actual end users, not just soleng users, no matter how I try to distance from `create` as long as `create` is on the command line.